cli/cmd/component-delete.go

```diff
@@ -105,17 +91,48 @@ func runDelete(cmd *cobra.Command, args []string) {
 		contextLogger.Fatal("Suitable kubeconfig file not found. Did you run 'lokoctl cluster apply' ?")
 	}
 
-	if err := deleteComponents(kubeconfig, componentsObjects...); err != nil {
+	if err := deleteComponents(kubeconfig, componentObjects); err != nil {
 		contextLogger.Fatal(err)
 	}
 }
 
-func deleteComponents(kubeconfig []byte, componentObjects ...components.Component) error {
+// componentNames returns list of components to operate on. If explicit list is empty,
+// it returns components defined in the configuration.
+func componentNames(list []string, lokomotiveConfig config.Config) []string {
```

Comment:
This function accepts more than it needs, so just pass: `RootConfig.Components` .

Reply:
Unfortunately, `config.component` is not exported, so I can only use `config.RootConfig` :/